Detectors/TPC/workflow/include/TPCWorkflow/LaserTracksCalibratorSpec.h

```diff
@@ -49,14 +49,12 @@ class LaserTracksCalibratorDevice : public o2::framework::Task
   void run(o2::framework::ProcessingContext& pc) final
   {
     const auto dph = o2::header::get<o2::framework::DataProcessingHeader*>(pc.inputs().get("laserTracks").header);
-    const auto startTime = dph->startTime;
-    const auto endTime = dph->startTime + dph->duration - 1;
 
+    o2::base::TFIDInfoHelper::fillTFIDInfo(pc, mCalibrator->getCurrentTFInfo());
     auto data = pc.inputs().get<gsl::span<TrackTPC>>("laserTracks");
-    LOGP(info, "Processing TF with start time {} and {} tracks", startTime, data.size());
+    LOGP(info, "Processing TF {} and {} tracks", mCalibrator->getCurrentTFInfo().tfCounter, data.size());
 
-    mCalibrator->getSlotForTF(startTime).getContainer()->setTFtimes(startTime, endTime);
```

Comment:
@wiechula why do you need to set these startTime and endTime (which are not really times but counters), I don't see where they are used. In any case, does not your container cover multiple TFs? Here the span of 1 event only is set.